src/atomic/mol.alert-msg/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { CSSProperties } from 'react';
+import { Style } from 'util';
+import { SpanProps, Span } from '../atm.span';
+
+interface AlertMsgProps extends SpanProps {
+  type: 'error' | 'warning' | 'info';
+}
+
+export function AlertMsg({type, children, ...rest}: AlertMsgProps) {
+
+  const resolveColorByType: any = (type: 'error' | 'warning' | 'info') => {
+    let style
+    switch(type) {
+      case 'error':
+        style = {color: 'red'};
+        break;
```

Comment:
aqui n daria pra usar o return direto?

```suggestion
    switch(type) {
      case 'error':
       return {color: 'red'};
```